src/api-serverless/src/tdh/api-tdh-routes.test.ts

```
@@ -0,0 +1,241 @@
+const mockFetchConsolidatedMetrics = jest.fn();
+
+jest.mock('../request-cache', () => ({
+  cacheRequest: () => (_req: unknown, _res: unknown, next: () => void) =>
+    next()
```

ESLint

[error] Delete `⏎···`

(prettier/prettier)